packages/xo-web/src/common/action-button.js

```diff
@@ -182,6 +182,17 @@ export default class ActionButton extends Component {
       </Button>
     )
 
-    return tooltip ? <Tooltip content={tooltip}>{button}</Tooltip> : button
+    return tooltip ? (
+      <Tooltip content={tooltip}>
+        {props.disabled ? (
+          // tooltip not showing up on disabled button
```

Comment:
Please take care of this comment https://github.com/vatesfr/xen-orchestra/pull/5038#discussion_r433857320